EA Sequences

jpaddison3 opened the pull request:
And change some fonts away from Times

The largest LW change here is the part where I noticed several times where LW was using Times as a default font. I thought in all those cases I'd rather the EA Forum use a sans serif stack. I introduce a uiSecondary typography, that LW sets to its serifStack and the EA Forum sets to its sansSerifStack.

I suggest using the sans serif stack for the Alignment Forum. I think it looks better in the context of the alignment forum theming. That's what I've put in the theme file, feel free to ask me to change.

![image (2)](https://user-images.githubusercontent.com/10352319/73881569-6d17a280-4815-11ea-8e3d-05aebd5e51ad.png)

Discordius commented on packages/lesswrong/components/posts/PostsPage/PostsAuthors.tsx:
```diff
@@ -7,7 +7,8 @@ const styles = createStyles(theme => ({
     display: 'inline',
-    ...theme.typography.postStyle
+    ...theme.typography.postStyle,
+    fontFamily: theme.typography.uiSecondary.fontFamily,
```
This shouldn't be here. `postStyle` already defined a fontFamily.

Discordius commented on packages/lesswrong/components/sequences/SequencesPage.jsx:
```diff
@@ -31,7 +31,8 @@ const styles = theme => ({
     fontVariant: "small-caps",
-    marginTop: 0
+    marginTop: 0,
+    fontFamily: theme.typography.uiSecondary.fontFamily,
```
Same here, if we want to add this here, we should get rid of the postStyle, which does nothing but define a fontFamily.

jpaddison3 replied:
I think we should, I think it's mildly more clear what's going on, and it allows the EA Forum (and AF?) to change to sans serif in places like this in one place.

Discordius replied:
Hmm, ok.

Discordius commented on packages/lesswrong/themes/eaTheme.js:
```diff
@@ -186,6 +189,16 @@ const theme = createLWTheme({
+        marginBottom: -8,
+      },
+      title: {
+        textTransform: 'lowercase',
```
Lowercase sure is a choice here. Surprised by that, but your call.

jpaddison3 replied:
It's then smallcapsed

Discordius replied:
Oh, I see.

jpaddison3 replied:
![image](https://user-images.githubusercontent.com/10352319/73882978-ee703480-4817-11ea-99bc-42d05f71f333.png)

Discordius replied:
Ah, I see. I think I prefer SmallCaps with variations for actual capital letters, but I can see the choice.

Raemon commented:
I'm merging since JP can't merge

jpaddison3 commented:
wait

jpaddison3 commented:
uh

jpaddison3 commented:
https://github.com/LessWrong2/Lesswrong2/pull/2898

jpaddison3 commented:
I think the first two were still reasonable comments, my "I think we should" referred to the block of Oli's conditional "if we want to add this here, we should ...". And I noticed a misalignment in the PostsPageActions.

Sorry, I should have mentioned that I was working on addressing the comments, I just had my lunch break.

Raemon commented:
Whoops. Sorry about that. (It's a bit tricky with non-LW-team-PRs, my preference is that PRs not be approved without actually being done because otherwise it's unclear what their state is, and there isn't any benefit to approving them earlier. That's more of an argument I'm having with Oli :P) 

Meanwhile, apologies for making it a bit more complicated. I think for now correct thing is to submit a new PR with the changes.

jpaddison3 commented:
Did so, already merged. Development velocity over 9000.
